app/src/main/java/com/infomaniak/mail/ui/main/folder/ThreadListAdapter.kt

```diff
@@ -435,7 +415,10 @@ class ThreadListAdapter @Inject constructor(
 
     private fun CardviewThreadItemBinding.displayAvatar(thread: Thread) {
         val (recipient, bimi) = thread.computeAvatarRecipient()
-        expeditorAvatar.loadAvatar(recipient, bimi)
+        expeditorAvatar.apply {
+            loadAvatar(recipient, bimi)
+            isFocusable = false
```

Comment:
Add a comment explaining that setting it inside the xml wouldn't work and why. Because I stumbled on it again and it still didn't make sens for a split second